### PR TITLE
Change instance to organization AB#28493

### PR DESCRIPTION
**Pull request by RubenGeo**
## Describe your changes

Brief description of your changes - not in-depth because the bulk of the description should be in the task on DevOps.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

**Comment by aberonni** [comment]
@RubenGeo can you rebase & solve the conflicts?

File: services/121-service/src/shared/enum/question.enums.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { LocalizedString } from '@121-service/src/shared/enum/language.enums';
```
Comment (aberonni) [inline services/121-service/src/shared/enum/question.enums.ts:1]: How was this working? 🤔  Could it be that this file isn't actually used anywhere?

File: services/121-service/src/seed-data/organization/organization-anonymous.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (aberonni) [inline services/121-service/src/seed-data/organization/organization-anonymous.json:1]: Looks like the deletion of the instance seed data slipped between the cracks

**Review by aberonni — CHANGES_REQUESTED** [review]

File: services/121-service/src/seed-data/organization/organization-anonymous.json (hunk 2 above)
Comment (RubenGeo) [reply services/121-service/src/seed-data/organization/organization-anonymous.json:1]: I don't get it? Was this file deleted in main? @aberonni

File: services/121-service/src/shared/enum/question.enums.ts (hunk 1 above)
Comment (RubenGeo) [reply services/121-service/src/shared/enum/question.enums.ts:1]: Not completely sure what you mean here. I think in the swc branch I put: LanguageEnum and LocalizedString in seperate files. Is that what you mean?

File: services/121-service/src/seed-data/organization/organization-anonymous.json (hunk 2 above)
Comment (aberonni) [reply services/121-service/src/seed-data/organization/organization-anonymous.json:1]: No, I mean that these seed files in `src/seed-data/organization` seem to be replacing the ones in `src/seed-data/instance` so I would assume that the latter folder would be deleted (but it doesn't seem that way).

File: services/121-service/src/shared/enum/question.enums.ts (hunk 1 above)
Comment (aberonni) [reply services/121-service/src/shared/enum/question.enums.ts:1]: You fixed an import in this file, which was broken. I don't understand how the broken import got onto main in the first place, and why there are no complaints about the broken import in CI. So I was wondering whether `question.enums.ts` is actually not imported anywhere and that is why there are no complaints. But just a theory/doubt

Comment (RubenGeo) [reply services/121-service/src/shared/enum/question.enums.ts]: Not sure how this happened, but `question.enum.ts` is imported in a bunch of files

File: services/121-service/src/seed-data/organization/organization-anonymous.json (hunk 2 above)
Comment (RubenGeo) [reply services/121-service/src/seed-data/organization/organization-anonymous.json]: Ah good one!

File: services/121-service/src/shared/enum/question.enums.ts (hunk 1 above)
Comment (aberonni) [reply services/121-service/src/shared/enum/question.enums.ts]: 🤷‍♂️ 😄

**Review by aberonni — APPROVED** [review]

**Pull request merged**